packages/nimble-components/src/select/index.ts

```diff
@@ -571,35 +585,44 @@ export class Select
                 if (!this.open) {
                     break;
                 }
+
+                this.setActiveOption(this.selectedIndex);
+
                 if (this.collapsible && this.open) {
                     e.preventDefault();
                     this.open = false;
                 }
 
-                if (this.selectedIndex !== this.indexWhenOpened!) {
-                    this.options[this.selectedIndex]!.selected = false;
-                    this.selectedIndex = this.indexWhenOpened!;
-                }
+                currentActiveIndex = this.selectedIndex;
                 this.focus();
                 break;
             }
             case keyTab: {
```

Comment:
I'm probably missing a nuance, but why do you need to handle the tab key here? What makes tab different than other events that lead to focusout?

Reply:
Nope. I think this was just a result of me creating an unneeded implementation for testing purposes. I've now changed the page object `pressTabKey` method to also emit a `focusout` event.